src/renderer/components/stake/withdraw/Withdraw.tsx

```diff
@@ -30,12 +39,28 @@ export const Withdraw: React.FC<Props> = ({
   selectedCurrencyAsset,
   runeShare,
   assetShare,
-  disabled
+  disabled: disabledProp,
+  fee,
+  updateFees
 }) => {
   const intl = useIntl()
-  const [withdrawPercent, setWithdrawPercent] = useState(50)
+  const [withdrawPercent, setWithdrawPercent] = useState(disabledProp ? 0 : 50)
+
+  const disabled = useMemo(() => withdrawPercent === 0 || disabledProp, [withdrawPercent, disabledProp])
 
   const withdrawAmounts = getWithdrawAmounts(runeShare, assetShare, withdrawPercent)
+  const fees = useMemo(
+    () => [
+      FP.pipe(
+        fee,
+        RD.map((fee) => ({
+          asset: BASE_CHAIN_ASSET,
+          amount: fee
+        }))
+      )
+    ],
+    [fee]
+  )
```

Comment:
The only reason to wrap a single feeRD into `[]` is the `Fee` component. So why not just do it directly in `<Fees fees={[fee]} />`. It will reduce other usage of `fees[0]`, too.

Reply:
yes. i thought about it but it will create NEW array for every render cycle which will lead to rerender Fees component

Reply:
Wrap it into a memo then. Maybe we should think about a `Fee` component for a single fee, then ;)

Reply:
@veado 
> Maybe we should think about a Fee component for a single fee, then ;)
i'd like to add this component to the AddStake and Send components in future to have a single way to manage fees rendering